torch/_inductor/config.py

```diff
@@ -157,10 +154,6 @@ class triton:
     convolution = "aten"
 
     # Always load full blocks (rather than broadcasting inside the block)
-    # Set default as True because otherwise will encouter `map::at` error
```

Comment:
Is this still an issue?

Reply:
It has been fixed.

Reply:
Then can we just remove this option?

Reply:
Yeah possibly. It still might be useful for debugging Triton issues.